From 3cfc6f1deda30359bcb1c8c075af2da9ff958c99 Mon Sep 17 00:00:00 2001
From: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
Date: Mon, 25 Nov 2019 01:42:00 +0000
Subject: [PATCH] arm64: cpu_errata: fix warning of return with a value in
 function 'void cpu_enable_trap_zva_access()'

Fix compile warning at arch/arm64/kernel/cpu_errata.c +103, in function 'cpu_enable_trap_zva_access'.

arch/arm64/kernel/cpu_errata.c:103:9: warning: 'return' with a value, in function returning void
return 0;
^

Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 arch/arm64/kernel/cpu_errata.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/kernel/cpu_errata.c b/arch/arm64/kernel/cpu_errata.c
index 60299c17d86e..22288cdd387c 100644
--- a/arch/arm64/kernel/cpu_errata.c
+++ b/arch/arm64/kernel/cpu_errata.c
@@ -100,7 +100,7 @@ static void cpu_enable_trap_zva_access(const struct arm64_cpu_capabilities *__un
 	else
 		config_sctlr_el2(SCTLR_EL1_DZE, 0);
 
-	return 0;
+	return;
 }
 #endif
 
-- 
2.17.1

